EKF/ekf.h

```diff
@@ -553,7 +551,7 @@ class Ekf : public EstimatorInterface
 	bool realignYawGPS();
 
 	// calculate the magnetic declination to be used by the alignment and fusion processing
```

Comment:
Now that this is returning the value, please ensure the header documentation indicates the units, eg radians

Reply:
Suggest:

// Returns the magnetic declination in radians to be used by the alignment and fusion processing.